pkg/client/transport/transport.go

```diff
@@ -63,8 +63,10 @@ func TLSConfigFor(c *Config) (*tls.Config, error) {
 	}
 
 	tlsConfig := &tls.Config{
-		// Change default from SSLv3 to TLSv1.0 (because of POODLE vulnerability)
-		MinVersion:         tls.VersionTLS10,
+		// Can't use SSLv4 because of POODLE and BEAST
```

Comment:
Oops good catch! I just created this PR to fix it https://github.com/kubernetes/kubernetes/pull/26562